docs/troubleshooting/authentication/custom_authentication_in_flutterFlow.md

```diff
@@ -0,0 +1,67 @@
+---
+keywords: ['authentication', 'api', 'configuration']
+slug: /custom-authentication-in-flutterflow
+title: Custom Authentication in FlutterFlow
+---
+# Custom Authentication in FlutterFlow
+
+:::info[Prerequisites]
+Ensure you have a **custom server** with login and sign-up endpoints that return a JWT token upon success
+
+**Custom authentication** must be enabled in FlutterFlow, with entry and logged-in pages correctly set
+:::
+
+
+Here's an example: 
+
+![](../assets/20250430121149388590.png)
+
+## Checklist for Troubleshooting
+
+### 1. Verify Server and API Endpoints
+    - Confirm that your server is correctly returning JWT tokens for login and sign-up requests. The server's response should include the authentication token, refresh token, expiration time, and user ID (UID).
+    - Double-check the API endpoint configurations in FlutterFlow to match your server's requirements.
+
+### 2. FlutterFlow Configuration
+    - Make sure custom authentication is enabled in the project settings.
+    - Verify that the entry point and logged-in pages are set correctly.
+
+### 3. UI Configuration
+    - To facilitate the authentication flow, ensure your app has at least three pages: 
+
+        **Login**
+
+        **Sign Up**
+
+        **Home Page** (i.e. the landing page when a user successfully authenticates)
```

Comment:
There is a bullet but its just one instruction, and then there is no bullets when we mention pages. Dont get the reasoning.